revscoring/scorers/util.py

```diff
@@ -0,0 +1,20 @@
+import numpy
+
+
+def normalize(v):
+    if isinstance(v, numpy.bool_):
+        return bool(v)
+    else:
+        return v
+
+def normalize_json(doc):
+    
+    if isinstance(doc, dict):
+        return {normalize_json(k):normalize_json(v) for k,v  in doc.items()}
```

Comment:
Does this work recursivelly? E.g. in objects like `{a:1, b:{c:0, d:2} }`, will 0 and 2 be treated as items?